### ISSUE TITLE
Document Import Blank After Entering FTP Information

### ISSUE THREAD

**Issue by TheWix** [thommcgrath/Beacon#205]
Hey there,

I have a server with GTX Gaming and I am trying to add my FTP server. I enter all the info and it finds the server no problem, but when I get to the screen  to "locate [my] Game.ini file" there is nothing listed: 
![image](https://user-images.githubusercontent.com/362449/82124561-c5900180-976d-11ea-8189-09c725b57092.png)

**Comment by thommcgrath** [comment]
Yeah Beacon’s FTP support doesn’t handle GTX Gaming. Their server sends data along the wrong channel for some reason.

Try Beacon 1.4, currently at alpha 2 as of this writing. You can get it from https://beaconapp.cc/download under the Preview Version section. While it does have bugs, updates are weekly, and it has brand new FTP code based on the curl library. It should be able to handle their FTP implementation.

**Comment by TheWix** [comment]
Hey, thanks for getting back to me so quickly!

I get this instantly when trying to connect with no way to close the window  without closing Beacon completely.

![image](https://user-images.githubusercontent.com/362449/82147197-0f372580-981c-11ea-9199-e1cbdd876e1a.png)

As an aside, do you accept outside pull requests?

**Comment by thommcgrath** [comment]
Wow, I’m surprised. For the mode, are you using Auto or one of the specific connection modes? I might have to start up a server with GTX again to see what could be going on.

I’ll look at any pull request, but wether or not I accept it really has everything to do with its content.

**Comment by thommcgrath** [comment]
Just an update on this. I confirmed that Beacon works with their service back in March. I rented server again to double check and something is wrong with their FTP now. It seems to be turned off. I've tried Beacon, an excellent FTP app on the Mac call Transmit, and even tried just opening a telnet tunnel to the listed FTP ip and port. This last attempt wouldn't allow me to log in as it's just a raw connection, so I'd have to write the protocol responses by hand if I got a connection. I couldn't. So it's not a username and password issue, it's not a protocol/mode issue, their server is outright refusing all connections of any kind of their listed address.

I'll open a support ticket with them, but best I can tell their firewall is blocking connections or their ftp service has crashed and nobody noticed. Or really any number of a hundred things.

Now I just need to make Beacon treat this as an error instead of getting stuck like it does now.

**Comment by thommcgrath** [comment]
Ok, final update to this. Turns out my test server was given a bad ip address, so nothing I could do would ever work. Once GTX figured this out, I was able to get a good connection. Beacon needs a small tweak to get it connecting to them, so 1.4 Alpha 3 will have the fix. Expect it out tomorrow.

**Comment by TheWix** [comment]
Great! I'll give it a test when it is released.
